packages/model-viewer/src/three-components/CachingGLTFLoader.ts

```diff
@@ -155,6 +172,10 @@ export class CachingGLTFLoader<T extends GLTFInstanceConstructor =
     this.dispatchEvent(
         {type: 'preload', element: element, src: url} as PreloadEvent);
     if (!cache.has(url)) {
+      if (meshoptDecoder) {
```

Comment:
We generally  use explicit combined null/undefined checks instead of truthiness: `if (meshoptDecoder != null)`.